oms-active-directory-security-audit-solution/azuredeploy.json

```diff
@@ -0,0 +1,768 @@
+            "customSolution": {
+                "name": "AD Security Audit Solution",
+                "displayName": "AD Security Audit Solution",
+                "LogAnalyticsApiVersion": "2017-03-03-preview"
+            }
+        },
+        "LogAnalyticsApiVersion": "2017-03-03-preview",
+        "SolutionVersion": "1.1",
+        "SolutionName": "AD Security Audit Solution",
+        "SolutionPublisher": "Asitha De Silva",
+        "ViewAuthor":  "asithahttp@gmail.com",
+        "ViewDescription":  "Active Directory Security Audit monitoring solution",
+        "ViewName":  "Users And Groups Monitoring",
+        "ViewName2":  "User Account Security Monitoring"
+        
+        },
+    
+
+"resources": [
+        {
+            "name": "[parameters('workspaceName')]",
+            "type": "Microsoft.OperationalInsights/workspaces",
+            "apiVersion": "2015-11-01-preview",
+            "location": "[parameters('workspaceRegionId')]",
+            "properties": {
+                "sku": {
+                    "name": "[parameters('pricingTier')]"
+                }
+            },
+
+    "resources": [
+        {
+            "name": "AD Security Audit Solution",
+            "location": "[parameters('workspaceRegionId')]",
+            "tags": { },
+            "type": "Microsoft.OperationsManagement/solutions",
+            "apiVersion": "2015-11-01-preview",
+            "dependsOn": [
+                "[concat('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+            ],
+            "properties": {
+                "workspaceResourceId": "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]",
+                "referencedResources": [
+                ],
+                "containedResources": [
+                    "[resourceId('Microsoft.OperationalInsights/workspaces/views/', parameters('workspaceName'), variables('ViewName'))]",
+                    "[resourceId('Microsoft.OperationalInsights/workspaces/views/', parameters('workspaceName'), variables('ViewName2'))]"
+                ]
+            },
+            "plan": {
+                "name": "AD Security Audit Solution",
+                "Version": "[variables('SolutionVersion')]",
+                "product": "MillenniumSolution",
+                "publisher": "[variables('SolutionPublisher')]",
+                "promotionCode": ""
+            }
+        },
+        {
+            "apiVersion": "[variables('LogAnalyticsApiVersion')]",
+            "name": "[concat(parameters('workspaceName'), '/', variables('ViewName'))]",
+            "type": "Microsoft.OperationalInsights/workspaces/views",
+            "location": "[parameters('workspaceregionId')]",
+            "id": "[Concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'),'/views/', variables('ViewName'))]",

```

Comment:
remove this

Reply:
i have two dashboards in my solution, i couldn't find a way to connect both views without this code. please give me a sample if you have, thanks